packages/api-explorer/__tests__/Result.test.jsx

```diff
@@ -46,6 +46,7 @@ describe('Result component', () => {
   })
 
   it('and render download file if isBinary is true', () => {
+    global.open = jest.fn();
```

Comment:
restore the original after the test finishes

Reply:
done